new/cookbook_quality_metrics.md

```diff
@@ -0,0 +1,48 @@
+---
+RFC: unassigned
+Author: Nathen Harvey <nharvey@chef.io>
+Status: Draft
+Type: Informational
+<Replaces: RFCxxx>
+<Tracking:>
+<  - https://github.com/chef/chef/issues/X>
+---
+
+# Cookbook Quality Metrics
+
+The question of what makes a good cookbook has been asked since the dawn of Chef.  As a community we've struggled with answering this in part because each recipe is unique to the specific requirements it is trying to meet.  Allowing Supermarket users to rate cookbooks is one thing that we've tried in the past.  This did not work because you simply do not have enough knowledge about the quality of a cookbook when you discover and download it from the Supermarket.  Similarly, download counts cannot be trusted because of [the way some older versions of the berkshelf-api server work](https://www.chef.io/blog/2015/01/21/those-pesky-supermarket-download-counts/).
+
+The Supermarket should make it easy for anyone to find quality cookbooks.  In order to do so, we must agree on some qualities that indicate whether or not a cookbook is considered to have high quality.  Ideally, these qualities are objective and able to be determined automatically.  These qualities should be defined and agreed on by the community.
+
+## Motivation
+
+    As a cookbook author,
+    I want to write cookbooks that can be shared with and easily used by the community
+    so that it is easier to get going with Chef.
+
+    As a Supermarket user,
+    I want to identify the relative quality of a given cookbook,
+    so that I can spend less time looking for a good cookbook and more time automating.
+
+## Specification
+
+A [Cookbook Quality Metrics](https://github.com/chef-cookbooks/cookbook-quality-metrics) system will be implemented to provide a quality score for each cookbook.  This score will be visible on the Supermarket.
+
+* The community will be able to collaborate on the metrics.
+* Each quality metric will add or remove points from a cookbook's score.
```

Comment:
I don't have super strong opinions on the numbering thing explicitly, but I do think that we need to have a simple and easy to view quality indicator - people aren't going to want to have to read and understand a bunch of flags and metrics when all they want is to install Apache.

What about something like they do with food packaging, where we calculate an aggregated "traffic light" score of Red, Amber or Green, with a breakdown of what composes that score (eg the nutritional information table) for those who want to dive in deeper?